qi-doc/scribblings/interface.scrbl

```diff
@@ -269,13 +278,16 @@ Each of the @racket[predicate] and @racket[consequent] expressions is a flow, an
              ...
              [else consequent ...])]
 )]{
-  Similar to @racket[lambda] but constrained to be a flow-based dispatcher. This is exactly equivalent to @racket[(lambda args (switch (args) maybe-divert-clause [predicate consequent ...] ... [else consequent ...]))]. @racket[switch-λ] and @racket[λ01] are aliases for @racket[switch-lambda].
+  Similar to @racket[lambda] but constrained to be a flow-based dispatcher. This is exactly equivalent to @racket[(lambda args (switch (args) maybe-divert-clause [predicate consequent ...] ... [else consequent ...]))] except ignoring the keywords. @racket[switch-λ] and @racket[λ01] are aliases for @racket[switch-lambda].
 
 @examples[
     #:eval eval-for-docs
-	((switch-lambda (x)
-	   [(and positive? odd?) (~> sqr add1)]
-	   [else _]) 5)
+    ((switch-lambda (a #:b b . a*)
+       [memq 'yes]
+       [else 'no]) 2 2 3 4 #:b 'any)
```

Comment:
nit: would be better to have examples that show how the keyword argument is _useful_ rather than how it is ignored. Maybe something like:

```
((switch-lambda (a #:fx fx . a*)
   [memq (~> 1> fx)]
   [else 'no]) 2 2 3 4 #:fx number->string)
```